bigtable/tests/unit/test_row.py

```diff
@@ -78,80 +78,65 @@ def _make_client(self, *args, **kwargs):
         table = object()
 
         row = self._make_one(row_key, table)
-        self.assertEqual(row._row_key, row_key_bytes)
+        self.assertEqual(row.row_mutations.mutations_entry.row_key, row_key)
         self.assertIs(row._table, table)
+        entry = bigtable_pb2.MutateRowsRequest.Entry()
+        self.assertEqual(row.row_mutations.mutations_entry.mutations,
+                         entry.mutations)
 
     def test_constructor_with_non_bytes(self):
         row_key = object()
         with self.assertRaises(TypeError):
             self._make_one(row_key, None)
 
-    def test__get_mutations(self):
+    def test_row_mutations(self):
         row_key = b'row_key'
         row = self._make_one(row_key, None)
 
-        row._pb_mutations = mutations = object()
-        self.assertIs(mutations, row._get_mutations(None))
+        row._row_mutations = mutations = object()
+        self.assertIs(mutations, row.row_mutations)
 
-    def _set_cell_helper(self, column=None, column_bytes=None,
-                         value=b'foobar', timestamp=None,
-                         timestamp_micros=-1):
-        import six
-        import struct
+    def _set_cell_helper(self, column=None, value=b'foobar', timestamp=None):
+
+        from google.cloud.bigtable.mutation import RowMutations
 
         row_key = b'row_key'
-        column_family_id = u'column_family_id'
+        column_family_id = 'column_family_id'
         if column is None:
             column = b'column'
         table = object()
+        row_mutations = RowMutations(row_key=row_key)
         row = self._make_one(row_key, table)
-        self.assertEqual(row._pb_mutations, [])
+        self.assertEqual(row.row_mutations.mutations_entry.row_key,
+                         row_mutations.mutations_entry.row_key)
         row.set_cell(column_family_id, column,
                      value, timestamp=timestamp)
 
-        if isinstance(value, six.integer_types):
-            value = struct.pack('>q', value)
-        expected_pb = _MutationPB(
-            set_cell=_MutationSetCellPB(
-                family_name=column_family_id,
-                column_qualifier=column_bytes or column,
-                timestamp_micros=timestamp_micros,
-                value=value,
-            ),
-        )
-        self.assertEqual(row._pb_mutations, [expected_pb])
+        expected_pb = row_mutations
+        self.assertEqual(row.row_mutations.mutations_entry.row_key,
+                         expected_pb.mutations_entry.row_key)
 
     def test_set_cell(self):
         self._set_cell_helper()
 
     def test_set_cell_with_string_column(self):
-        column_bytes = b'column'
         column_non_bytes = u'column'
-        self._set_cell_helper(column=column_non_bytes,
-                              column_bytes=column_bytes)
+        self._set_cell_helper(column=column_non_bytes)
 
     def test_set_cell_with_integer_value(self):
-        value = 1337
+        value = b'1337'
```

Comment:
If we don't support passing integer values, then just delete this test altogether.

Reply:
@aneepct ping